projects/plugins/boost/app/assets/src/js/pages/index/index.tsx

```diff
@@ -116,6 +124,22 @@ const Index = () => {
 			>
 				<CloudCssMeta />
 			</Module>
+			<Module
+				slug="page_cache"
+				title={ __( 'Cache Site Pages', 'jetpack-boost' ) }
+				description={
+					<p>
+						{ __(
+							'Store and serve preloaded content to reduce load times and enhance your site performance and user experience.',
+							'jetpack-boost'
+						) }
+					</p>
+				}
+				onEnable={ invalidatePageCacheError }
+				onDisable={ invalidatePageCacheError }
```

Comment:
Is it worth invalidating on disable?

Reply:
Yeah. Otherwise, if there was an error when you enabled the module and then disabled it, when you enable again, you'd see the error appear for a sec and disappear, since the error store hasn't been updated yet.

I feel like this is a band-aid for a problem I haven't pinpointed yet.